kubernetes/structures_container.go

```diff
@@ -714,6 +717,10 @@ func expandContainerVolumeMounts(in []interface{}) ([]v1.VolumeMount, error) {
 		if subPath, ok := p["sub_path"]; ok {
 			vmp[i].SubPath = subPath.(string)
 		}
+		if mountPropagation, ok := p["mount_propagation"]; ok {
+			temp := v1.MountPropagationMode(mountPropagation.(string))
```

Comment:
```suggestion
			mp := v1.MountPropagationMode(mountPropagation.(string))
```
It's preferable not to have generic variables names like `temp`. It helps at least when grep-ing the code.